scripts/setup/pre_reboot/6_setup_logging.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
-source ${PROJECT_PATH}/config/scripts_config.sh
+# This tells the shell to exit the script if any command within the script exits with a non-zero status
+set -e
 
-touch ${LOGROTATE_PATH}/${LOGCONF_FILENAME}
+# Setup logrotate config file
+touch ${LOGROTATE_PATH}/${LOGCONF_FILENAME} > /dev/null
```

Comment:
Redirection is unnecessary for `touch`. No output

Reply:
Removed.